Sources/SwiftGRPC/Runtime/ClientCallUnary.swift

```diff
@@ -56,3 +56,10 @@ open class ClientCallUnaryBase<InputType: Message, OutputType: Message>: ClientC
     return self
   }
 }
+
+/// Simple fake implementation of `ClientCallUnary`.
+open class ClientCallUnaryTestStub<InputType: Message, OutputType: Message>: ClientCallUnary {
```

Comment:
Does this need to be generic at all?

Reply:
Not strictly necessary at the moment, was primarily going for consistency with `ClientCallClientStreamingTestStub`.  The generics would be necessary if we want similar request queueing or response recording, thought TBH I find that approach a bit to constrained for my use-cases. In unit tests I'd generally want to test a lot of error cases and in integration tests I'd generally want the stub to have business logic (as in, I really want an in-memory fake).

I don't have strong preferences regarding the generics, let me know if you'd prefer I remove them. Happy to update the PR either way.

Reply:
Thanks for the elaboration! For now, let's remove them; I'm open to re-adding them later on if necessary.

Reply:
Done.